test/integration/targets/vmware_inventory/test_vmware_inventory.yml

```diff
@@ -0,0 +1,27 @@
+# Test code for the vmware dynamic plugin module
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+---
+- name: Test VMware Dynamic Inventroy Plugin
+  hosts: localhost
+  tasks:
+    - name: make sure pyvmomi is installed
+      pip:
+        name: pyvmomi
+        state: latest
+      when: "{{ ansible_user_id == 'root' }}"
```

Comment:
This is redundant with the install in runme.sh.

Reply:
ok.